src/compiler/diagnosticMessages.json

```diff
@@ -1775,6 +1775,14 @@
         "category": "Error",
         "code": 2661
     },
+    "Cannot find name '{0}'. Did you mean to prefix the static member with the class name, '{1}.{0}'?": {

```

Comment:
I agree, the shorter form sounds better. The `this` version can be even shorter: "Did you mean the memer 'this.{0}'?"